### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Shaq-newell-wtc**

### CODE REVIEW

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="FeedingHopeAndHeroes.css" />
+    <script src="https://kit.fontawesome.com/yourcode.js"></script>
```
Comment (anthonydillon) [inline FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:5]: Could you add a `title` please

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      </div>
+    </div>
+
+    <div class="donation-form-section">
+      <h1 class="donation-form-header">
+        If you are a food supplier or restaurant who would like to donate
+      </h1>
+      <div class="form-container">
+        <div class="form-left">
+          <form>
```
Comment (anthonydillon) [inline FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:64]: Does this form have an action to post too?

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+              <label class="donation-label" for="message">Message</label><br />
+              <textarea
+                type="text"
+                class="message"
+                name="message"
+                value=""
+              /></textarea><br />
+              <button type="submit" class="donation-button form-submit-button">
+                Thank you
+              </button>
```
Comment (anthonydillon) [inline FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:104]: Maybe add a small piece of text explaining what the submitter should expect next. Phone call? Email? Site visit? and when? In the next 2 days or so?

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    <div class="carousel-section">
+      <h1 class="carousel-header">
+        Follow us and share your experiences on Social Media </br>
+        @feedinghopeandheroes
+      </h1>
+      <div class="carousel-container">
+
+      </div>
+      <div class="follow-button-container">
+        <button class="follow-button">
```
Comment (anthonydillon) [inline FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:177]: Should this be a link?

**Review by anthonydillon — COMMENTED** [review]
Some small comments. If you would like us to address them please let us know.

**Review by pmahnke — APPROVED** [review]
approving

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html (hunk 3 above)
Comment (Shaq-newell-wtc) [reply FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:104]: This is a placeholder made while waiting for final design

File: FeedingHopeAndHeroes/FeedingHopeAndHeroes.html (hunk 4 above)
Comment (Shaq-newell-wtc) [reply FeedingHopeAndHeroes/FeedingHopeAndHeroes.html:177]: This is also a placeholder until more details are confirmed

**Pull request merged**